pmg/views.py

```diff
@@ -76,6 +76,27 @@ def classify_attachments(files):
 
     return audio, related
 
+@app.context_processor
+def inject_user_following():
+    user_following = []
+
+    if current_user.is_authenticated():
+
+        # Append user-followed committees if logged in
+        user_following = current_user.following
+
+        for committee in user_following:
+            meetings = load_from_api('v2/committees/%s/meetings' % committee.id,
+                fields=['id','title','date'])['results']
+
+            if len(meetings):
+                setattr(committee, 'recent_meeting', meetings[0])
```

Comment:
can't do that -- committee is a sqlalchemy object, which cannot be assigned to directly